examples/mysql-galera/image/my.cnf.perc/my.cnf

```diff
@@ -0,0 +1,125 @@
+#
```

Comment:
I need to remove this. This was a duplicate when I brought the code over from my own repo.